apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter.css

```diff
@@ -0,0 +1,27 @@
+table {
```

Comment:
There's not much CSS here, but in the future, be sure to tag Justin for reviews on the CSS in PRs.